index.html

```diff
@@ -120,17 +120,18 @@ <h2 class="h5">Qui peut se faire vacciner contre la Covid-19&nbsp;?*</h2>
                     De nombreuses personnes sont éligibles à la vaccination contre la Covid-19. On peut être éligible à la vaccination selon son âge, sa santé, ou sa profession. Voici la liste des personnes qui peuvent dès à présent être vaccinées&nbsp;:
                 </p>
                 <ul>
-                    <li>personnes de 50 ans et plus quel que soit leur lieu de vie et leur état de santé (avec ou sans comorbidités)&nbsp;;</li>
-                    <li>personnes de 16 à 49 ans inclus souffrant d’une pathologie à très haut risque de forme grave de Covid-19&nbsp;;</li>
-                    <li>personnes de 18 à 49 ans inclus souffrant de comorbidités&nbsp;;</li>
-                    <li>personnes majeures en situation de handicap, hébergées en maison d’accueil spécialisée ou foyer d’accueil médicalisé&nbsp;;</li>
+                    <li>personnes de plus de 50 ans, quel que soit leur lieu de vie et leur état de santé (avec ou sans comorbidités)&nbsp;;</li>
+                    <li>personnes de plus de 18 ans souffrant de comorbidités&nbsp;;</li>
```

Comment:
@Luwangel je me demande s'il ne faudrait pas mettre le dernier point en tant "précision" de cette ligne

Quelque chose comme : 
```suggestion
                    <li>personnes de plus de 18 ans souffrant de comorbidités&nbsp;;<br/>
                    <span class="fw-bold">À partir du 31 mai 2021</span>: toute personne de plus de 18 ans (avec ou sans comorbidités) sera éligible à la vaccination.
                    </li>
```

Reply:
@fcamblor Je pensais que les parenthèses étaient assez claires. Je te laisse faire la modif si tu penses que c'est nécessaire (je n'ai pas mon ordi sous la main ce week-end pour vérifier le rendu). Par contre autant valider vite avant que le 31 mai n'arrive :)

Reply:
J'ai finalement supprimé cette ligne qui était incluse dans la ligne sur les personnes de plus de 16 ans.